conda/seacas/meta.yaml

```diff
@@ -9,8 +9,9 @@ package:
   version: {{ seacas_version }}
 
 source:
-  git_url: https://github.com/sandialabs/seacas
-  git_rev: {{ seacas_git_rev }}
+  - git_url: https://github.com/sandialabs/seacas
+  - git_rev: {{ seacas_git_rev }}
```

Comment:
Maybe this is no longer grabbing the correct versions. Investigate on Monday if latest push still fails.

Reply:
Addressed in later commits.